apps/web/components/dashboard/admin/ServerStats.tsx

```diff
@@ -0,0 +1,133 @@
+  }
+  return (
+    <div className="text-nowrap">
+      <span className="text-3xl font-semibold">{currentRelease}</span>
+      <span className="ml-1 text-sm">{newRelease}</span>
+    </div>
+  );
+}
+
+export default function ServerStats() {
+  const { data: serverStats } = api.admin.stats.useQuery(undefined, {
+    refetchInterval: 1000,
+    placeholderData: keepPreviousData,
+  });
+
+  if (!serverStats) {
+    return <LoadingSpinner />;
+  }
+
+  return (
+    <>
+      <div className="mb-2 text-xl font-medium">Server Stats</div>
+      <div className="flex flex-col gap-4 sm:flex-row">
+        <div className="rounded-md border bg-background p-4 sm:w-1/4">
+          <div className="text-sm font-medium text-gray-400">Total Users</div>
+          <div className="text-3xl font-semibold">{serverStats.numUsers}</div>
+        </div>
+        <div className="rounded-md border bg-background p-4 sm:w-1/4">
+          <div className="text-sm font-medium text-gray-400">
+            Total Bookmarks
+          </div>
+          <div className="text-3xl font-semibold">
+            {serverStats.numBookmarks}
+          </div>
+        </div>
+        <div className="rounded-md border bg-background p-4 sm:w-1/4">
+          <div className="text-sm font-medium text-gray-400">
+            Server Version
+          </div>
+          <ReleaseInfo />
+        </div>
+      </div>
+
+      <div className="flex flex-col sm:flex-row sm:gap-8">
+        <div className="sm:w-1/2">
+          <div className="mb-2 mt-8 text-xl font-medium">Background Jobs</div>
+          <Table className="rounded-md border">
+            <TableHeader className="bg-gray-200">
+              <TableHead>Job</TableHead>
+              <TableHead>Queued</TableHead>
+              <TableHead>Pending</TableHead>
+              <TableHead>Failed</TableHead>
+            </TableHeader>
+            <TableBody>
+              <TableRow>
+                <TableCell className="lg:w-2/3">Crawling Jobs</TableCell>
+                <TableCell>{serverStats.crawlStats.queuedInRedis}</TableCell>
+                <TableCell>{serverStats.crawlStats.pending}</TableCell>
+                <TableCell>{serverStats.crawlStats.failed}</TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell>Indexing Jobs</TableCell>
+                <TableCell>{serverStats.indexingStats.queuedInRedis}</TableCell>
+                <TableCell>-</TableCell>
+                <TableCell>-</TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell>Inference Jobs</TableCell>
+                <TableCell>
+                  {serverStats.inferenceStats.queuedInRedis}
+                </TableCell>
+                <TableCell>{serverStats.inferenceStats.pending}</TableCell>
+                <TableCell>{serverStats.inferenceStats.failed}</TableCell>
+              </TableRow>
+            </TableBody>
+          </Table>
+        </div>
+        <AdminActions />
```

Comment:
Not sure I like `AdminActions` here, can we extract it to the root page?

Reply:
Done, moved it to the root page, it changes the layout a bit. But since it's admin page, we can keep it like that.